app/controllers/RatingsController.scala

```diff
@@ -11,11 +11,11 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class RatingsController @Inject()(cc: ControllerComponents,
                                   actorSystem: ActorSystem,
-                                  ratingStats: RatingStats)
+                                  ratingStats: RatingStats,
+                                  redisStoreClient: RedisStoreClient)
                                  (implicit exec: ExecutionContext) extends AbstractController(cc) {
   def ratingInfo(movieId: Int): Action[AnyContent] = Action.async { _ =>
-    val redisHandler = new RedisStoreClient
-    val median = redisHandler.getMedian
+    val median = redisStoreClient.getMedian
```

Comment:
can you give me some reference about Median?

Reply:
Sent email with instructions / requirements and here is a general median description: https://en.wikipedia.org/wiki/Median
Let me know if you want more clarification